src/test/CMakeLists.txt

```diff
@@ -7,7 +7,6 @@ add_subdirectory(abi_component)
 
 if (WIN32)
     add_subdirectory(component)
-    add_subdirectory(cpp)
```

Comment:
We don't have C++ tests anymore?

Reply:
Not universal tests no. We have cppwinrt-specific tests only. Once Ryan has the xlang version spun up there will likely be xlang-specific C++ tests for that.